addons/website_sale/views/templates.xml

```diff
@@ -224,13 +224,15 @@
 
     <template id="sort" inherit_id="website_sale.products" active="True" customize_show="True" name="Show Sort by">
         <xpath expr="//div[@class='products_pager']/div/t[@t-call][last()]" position="after">
-            <t t-set="list_price_desc_label">Price <i class="fa fa-caret-up" aria-hidden="true"/></t>
-            <t t-set="list_price_asc_label">Price <i class="fa fa-caret-down" aria-hidden="true"/></t>
-            <t t-set="name_asc_label">Name <i class="fa fa-caret-down" aria-hidden="true"/></t>
+            <t t-set="list_price_desc_label">Price - High to Low</t>
+            <t t-set="list_price_asc_label">Price - Low to High</t>
```

Comment:
According to https://github.com/odoo/odoo/commit/4f88f4dd18cb3fd10b5b40d69241a04c36359269 and a discussion with @fgi-odoo, can you use those labels:
- Catalog price: high to low
- Catalog price: low to high